exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,18 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
```

Comment:
Function works correctly but some of the conditions in the if statements are not needed. Try to have a look at which ones might not be need to simplify logic here.